.travis.yml

```diff
@@ -43,9 +42,13 @@ jobs:
     - pipenv run pytest -v -m integration
   - stage: Deploy to dev
     script:
+    - aws sqs create-queue --queue-name timereport-slack-command-dev
+    - aws sqs set-queue-attributes --region eu-north-1 --queue-url https://sqs.eu-north-1.queue.amazonaws.com/106618949447/timereport-slack-command-dev --attributes VisibilityTimeout=60
```

Comment:
This queue-url thingy feels like it could be a potential problem.
Would be nice to be able to change maybe the region and organisation? Also the name of the queue should probably be an variable to make sure we use the same name in both commands?

Or should we leave it like this and do it more dynamic (env vars etc) in the future if necessary?

Reply:
Fixed by moving the attribute to creation of the queue. Then we don't need to manage the full url at all.